src/classes/Util.js

```diff
@@ -183,11 +183,15 @@ class Util {
   }
 
   static getEmoji(d, Emoji) {
+    if (Emoji.includes(":")) {
+      Emoji = Emoji.split(":")[2].split(">")[0];
+    }
+
     return d.client.emojis.cache.find(
-        (x) =>
-            x.name.toLowerCase() === Emoji.toLowerCase().addBrackets() ||
-            x.id === Emoji ||
-            x.toString() === Emoji
+      (x) =>
+        x.name.toLowerCase() === Emoji.toLowerCase().addBrackets() ||
+        x.id === Emoji ||
+        x.toString() === Emoji
```

Comment:
`d.client.emojis.resolve(Emoji)` to directly get GuildEmoji | null

Reply:
Tried this, doesn't work as intended.

Only works when directly given the id of the emoji

Reply:
hmm then we can leave it ig